app/decorators/hearing_decorator.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class HearingDecorator < BaseDecorator
+  def provider_list
+    return t('generic.not_available') if providers.blank?
+
+    safe_join(provider_sentences, tag.br)
```

Comment:
We should agree a consistent approach for separators, have asked Dan Frampton if he has an opinion.

Reply:
tangent at this point. separate PR

Reply:
👍